app/design/frontend/yemelyanov/eag_theme/Magento_Theme/layout/default.xml

```diff
@@ -7,24 +7,14 @@
                 <argument name="logo_img_height" xsi:type="number">200</argument>
             </arguments>
         </referenceBlock>
-
-
-        <referenceContainer name="footer">
-            <container name="footer.geekhub.container" htmlClass="pseudo-container" htmlTag="div" after="footer_links">
-
-                <block class="Magento\Framework\View\Element\Text" name="geekhub.label" as="geekhub_label">
-                    <action method="setText">
-                        <argument xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="value" xsi:type="string">This text from block geekhub.label in footer.geekhub.container</argument>
-                    </action>
-                </block>
-
-                <block class="Magento\Framework\View\Element\Template" name="geekhub.template" as="geekhub_template_block" template="Magento_Theme::eag/footer_item_template.phtml"/>
-
-            </container>
-        </referenceContainer>
-
-
-    </body>
-</page>
+                <referenceContainer name="sidebar.additional">
```

Comment:
Text indentation is not correct (number of space characters)
<img src="http://dl3.joxi.net/drive/2018/11/02/0021/1588/1439284/84/e43fe2d344.png">